prismic-model/src/exhibition-guides.ts

```diff
@@ -0,0 +1,44 @@
+import { CustomType } from './types/CustomType';
+import title from './parts/title';
+import link from './parts/link';
+import list from './parts/list';
+import image from './parts/image';
+import structuredText from './parts/structured-text';
+import embed from './parts/embed';
+import contributorsWithTitle from './parts/contributorsWithTitle';
+import number from './parts/number';
+
+const exhibitionGuides: CustomType = {
+  id: 'exhibition-guides',
+  label: 'Exhibition Guide',
+  repeatable: true,
+  status: true,
+  json: {
+    Guide: {
+      title,
+      'related-exhibition': link('Related Exhibition', 'document', [
+        'exhibitions',
+      ]),
+    },
+    // We are providing a repeatable list of guide components which could be:
+    // A gallery section, a subsection, or a stop within those sections
+    // We did have an extra field 'partOf' where editors can indicate what section or subsection
+    // a stop is related to, but removed this to get a first iteration and think about hierarchy structure later
+    Component: {
```

Comment:
also think this should be plural, it's the tab the editors will see in the Prismic UI